commit 416a4b77fedd2c117a7d695f6141e3865cf4bd55
Author: Michael Still <mikal@stillhq.com>
Date:   Thu Aug 6 11:13:31 2026 +1000

    Use a routable IP for qemu SPICE consoles.
    
    We currently do this is the SPICE HTML5 transcoding
    proxy, but spice-direct consoles need it as well. We
    should therefore simply always configure it.
    
    Partial-Bug: #2131793 (spice-direct consoles)
    Change-Id: Ic88a58d974b71e38bc91fd7dee9eef38a0bacb87
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/roles/nova-cell/templates/nova.conf.j2 b/ansible/roles/nova-cell/templates/nova.conf.j2
index bd7359956..2b02fd6ed 100644
--- a/ansible/roles/nova-cell/templates/nova.conf.j2
+++ b/ansible/roles/nova-cell/templates/nova.conf.j2
@@ -87,11 +87,11 @@ jpeg_compression = {{ nova_spice_jpeg_compression }}
 zlib_compression = {{ nova_spice_zlib_compression }}
 playback_compression = {{ nova_spice_playback_compression }}
 streaming_mode = {{ nova_spice_streaming_mode }}
+server_proxyclient_address = {{ api_interface_address }}
 {%     if enable_kerbside | bool %}
 spice_direct_proxy_base_url = "https://{{ kolla_external_fqdn }}:13002/nova-console.vv"
 {%     endif %}
 {%     if nova_spice_html5 | bool %}
-server_proxyclient_address = {{ api_interface_address }}
 {%         if inventory_hostname in groups[nova_cell_compute_group] %}
 html5proxy_base_url = {{ nova_spicehtml5proxy_fqdn | kolla_url(public_protocol, nova_spicehtml5proxy_public_port, '/spice_auto.html') }}
 {%         endif %}
